analyses/compare-gistic/02-GISTIC-tidy-data-prep.Rmd

```diff
@@ -96,6 +96,85 @@ lgat_del_genes_file <- file.path(lgat_gistic_dir, "del_genes.conf_90.txt")
 
 ## Generate gene-level tidy tables
 
+### Explore amp and del genes data 
```

Comment:
Don't forget to get rid of this exploratory section. I don't think we need it now.